src/items/rules/resolve_points.json

```diff
@@ -0,0 +1,191 @@
+        "format": 1
+      },
+      "title": {
+        "level": 1,
+        "show": true
+      },
+      "video": {
+        "controls": true,
+        "volume": 0.5
+      }
+    },
+    {
+      "_id": "Vxnd8TIaaGDlGVXZ",
+      "name": "Utility Specialized Uses",
+      "type": "text",
+      "_stats": {
+        "compendiumSource": null,
+        "coreVersion": "12.331",
+        "createdTime": 1734732193952,
+        "duplicateSource": null,
+        "lastModifiedBy": "Ze895EIq7nm5kMrK",
+        "modifiedTime": 1734732242696,
+        "systemId": "sfrpg",
+        "systemVersion": "0.27.0"
+      },
+      "flags": {},
+      "image": {},
+      "ownership": {
+        "Ze895EIq7nm5kMrK": 3,
+        "default": -1
+      },
+      "sort": 500000,
+      "src": null,
+      "system": {},
+      "text": {
+        "content": "<p>This category’s uses of Resolve Points can be useful in a pinch.</p><h2>Exceptional Aid:</h2><p>When you attempt to aid another on a skill check, you can spend 1 Resolve Point to roll your skill check twice and use the higher result. If you succeed, the creature you’re aiding gains a +3 bonus to their skill check, rather than the +2 you would usually grant.</p><h2>Just the Thing:</h2><p>As a standard action, you can reach into your backpack, pocket, or other suitable storage space, and spend Resolve Points to acquire an item you can afford but don’t own. The item must have a bulk of 1 or less, and an item level no greater than your level – 1. You immediately lose a number of credits equal to the value of the item, as this ability assumes you prepurchased or traded for this item at some point in the past. If the item you pull out is a consumable item (such as a grenade, serum, or spell gem), you must spend 1 Resolve Point. Otherwise, you must spend a number of Resolve Points equal to half the item level.</p><h2>Quick Fusion:</h2><p>When you apply a fusion seal to a weapon, you can spend Resolve Points to reduce the time it takes for that fusion to function. If you spend 1 Resolve Point, that fusion seal functions after 10 minutes rather than the normal 24 hours. If you spend 2 Resolve Points, that fusion seal functions immediately after you transfer it.</p><h2>Rapid Moves:</h2><p>You can spend 1 Resolve Point to take an additional move action on your turn. You can’t use this ability on a round that you take a full action.</p><h2>Skilled Hacker:</h2><p>When you take an additional major action during a dynamic hacking encounter (Starfinder Tech Revolution 70), you can spend 1 Resolve Point to ignore the cumulative penalty that taking that additional action would impose on your skill checks for the turn.</p>",
```

Comment:
Change "(Starfinder Tech Revolution 70)" to "(Tech Revolution pg. 70)"

Reply:
No problem making the change, but I just copied the text in from the book.....

Reply:
Fair point, actually, probably best to leave it as written.